src/ontology/mondo-edit.obo

```diff
@@ -405892,10 +405892,10 @@ synonym: "del(8)" EXACT [NCIT:C36535]
 synonym: "Deletions of chromosome 8" RELATED [MESH:C537823]
 synonym: "loss of chromosome 8" EXACT [NCIT:C36535]
 xref: GARD:0012070 {source="MONDO:equivalentTo"}
-xref: MESH:C537823 {source="MONDO:equivalentTo"}
-xref: NCIT:C36535 {source="MONDO:equivalentTo"}
+xref: MESH:C537823
```

Comment:
we should have some sort of relationship here, you could do MONDO:relatedTo if they are not exact
(also applies to NCIT xref below)

Reply:
I just fixed this because there were conflicts (fixed this conflict and this too)